Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -126,9 +126,9 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
     private lazy var adapter: WireGuardAdapter = {
         WireGuardAdapter(with: self) { logLevel, message in
             if logLevel == .error {
-                os_log("🔵 Received error from adapter: %{public}@", log: .networkProtection, type: .error, message)
+                os_log("🔴 Received error from adapter: %{public}@", log: .networkProtection, type: .error, message)
```

Comment:
I'd still love to do something more with this at some point, but definitely out of scope for this PR.